src/Command/ProductExportCommand.php

```diff
@@ -128,12 +170,56 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
     private function getFeedColumns(): array
     {
-        $base = (array) $this->container->getParameter('factfinder.export.columns.base');
-        return array_values(array_unique(array_merge($base, array_map([$this, 'getFieldName'], $this->productFields))));
+        $baseFieldNames    = (array) $this->container->getParameter('factfinder.export.columns.base');
+        $productFieldNames = $this->getProductFieldNames($this->productFields);
+
+        dd($productFieldNames);
+        return array_values(array_unique(array_merge($baseFieldNames, $productFieldNames)));
+    }
+
+    private function getProductFieldNames(array $productFields): array
+    {
+        $fields = [];
+
+        /** @var FieldInterface $productField */
+        foreach ($productFields as $productField) {
+            if ($productField instanceof Price) {
+                foreach ($this->getCurrencyList() as $currency) {
+                    $fields[] = $currency['factor'] === $this->getDefaultCurrency()->getFactor()
+                        ? $productField->getName()
+                        : $productField->getName() . '_' . $currency['iso_code'];
+                }
+            }
+
+            $fields[] = $productField->getName();
+        }
+
+        return $fields;
     }
 
-    private function getFieldName(FieldInterface $field): string
+    private function setCurrencyList(): self
     {
-        return $field->getName();
+        $query = $this->connection->createQueryBuilder()
+            ->select('iso_code, factor')
+            ->from('currency')
+            ->execute()
+        ;
+
+        $this->currencyList = $query->fetchAllAssociative();
+
+        return $this;
+    }
+
+    private function setDefaultCurrency(): self
+    {
+        $this->defaultCurrency = $this->currencyRepository->search(
+            (new Criteria())->addFilter(new EqualsFilter('factor', 1)),
+            new Context(new SystemSource())
+        )
+            ->getElements();
```

Comment:
formatting

Reply:
according to php-cs-fixers everything is fine

Reply:
cs fixer is not the absolute wisdom we should trust i'd say. Isn't this 
`->getElements()`
in next line look weird to you?